pyproject.toml

```diff
@@ -150,7 +150,7 @@ geoips = "geoips"
 convert_trackfile_to_yaml = "geoips.commandline.convert_trackfile_to_yaml:main"
 create_sector_image = "geoips.commandline.create_sector_image:main"
 list_available_plugins = "geoips.commandline.list_available_plugins:main"
-run_procflow = "geoips.commandline.run_procflow:main"
+run_procflow = "geoips.commandline.commandline_interface:main"
```

Comment:
Have we made the same update to `pyproject.toml` from `data_fusion`?

Reply:
Yup! See NRLMMD-GEOIPS/data_fusion#31.